syntax/entry.php

```diff
@@ -296,18 +298,23 @@ function replaceQuery() {
      * @param Doku_Renderer_plugin_data_edit $renderer
      */
     function _editData($data, &$renderer) {
-        $renderer->form->startFieldset($this->getLang('dataentry'));
-        $renderer->form->_content[count($renderer->form->_content) - 1]['class'] = 'plugin__data';
-        $renderer->form->addHidden('range', '0-0'); // Adora Belle bugfix
+        $renderer->form->addFieldsetOpen($this->getLang('dataentry'));
+        $renderer->form->getElementAt($renderer->form->elementCount() - 1)->attr('class', 'plugin__data');
```

Comment:
I wonder if we can be more robust here now. Instead of relying on the last element being the correct one, use one of the findElement* methods

Reply:
Fixed. This was a strange way to add a class to an element that was just created. It can be done in one go.